p2gccCPack.cmake

```diff
@@ -0,0 +1,39 @@
+# DEB and RPM disabled by default, but can be built manually via `cpack -G DEB`
+set(CPACK_GENERATOR
+    TGZ
+    ZIP
+)
+
+set(CPACK_INCLUDE_TOPLEVEL_DIRECTORY 0)
+set(CPACK_INSTALL_CMAKE_PROJECTS
+    "${CMAKE_BINARY_DIR}"
+    "${PROJECT_NAME}"
+    ALL
+    .
+)
+
+set(CPACK_PACKAGE_VENDOR "Dave Hein")
+set(CPACK_PACKAGE_CONTACT "Dave Hein <davehein@swbell.net>")
+set(CPACK_PACKAGE_VERSION_MAJOR ${PROJECT_VERSION_MAJOR})
+set(CPACK_PACKAGE_VERSION_MINOR ${PROJECT_VERSION_MINOR})
+set(CPACK_PACKAGE_VERSION_PATCH ${PROJECT_VERSION_PATCH})
+set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH})
+if (PROJECT_VERSION_TWEAK)
+    set(CPACK_PACKAGE_VERSION ${CPACK_PACKAGE_VERSION}.${PROJECT_VERSION_TWEAK})
+endif ()
+set(CPACK_PACKAGE_RELOCATABLE ON)
+
+# Debian Specific
+set(CPACK_DEBIAN_PACKAGE_SECTION                    devel)
+set(CPACK_DEBIAN_PACKAGE_PRIORITY                   optional)
+set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS                  ON)
+set(CPACK_DEBIAN_PACKAGE_RELEASE                    1)
+set(CPACK_DEBIAN_FILE_NAME                          DEB-DEFAULT)
+set(CPACK_DEBIAN_PACKAGE_CONTROL_STRICT_PERMISSION  ON)
+
+# RPM Specific
+set(CPACK_RPM_FILE_NAME                         RPM-DEFAULT)
+set(CPACK_RPM_PACKAGE_LICENSE                   "MIT")
```

Comment:
I couldn't find any reference to a license, so I was guessing you wanted it MIT-licensed since that is what most of the Parallax community goes with.